eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -91,7 +103,7 @@ class Peer(nodeParams: NodeParams, remoteNodeId: PublicKey, authenticator: Actor
         case Some((gf, lf)) => wire.Init(globalFeatures = gf, localFeatures = lf)
         case None => wire.Init(globalFeatures = nodeParams.globalFeatures, localFeatures = nodeParams.localFeatures)
       }
-      log.info(s"using globalFeatures=${localInit.globalFeatures.toBin} and localFeatures=${localInit.localFeatures.toBin}")
+      log.debug(s"using globalFeatures={} and localFeatures={}", localInit.globalFeatures.toBin, localInit.localFeatures.toBin)
```

Comment:
This log is useful as information because there can be peer specific params. Ok for the formatting changes but then you also need to update line 138. As a rule of thumb I mostly keep the `{}` notation for debug statements on hot paths.